README.rst

```diff
@@ -0,0 +1,93 @@
+============================================
+Bugzilla: A Free and Open-Source Bug Tracker
+============================================
+
+Bugzilla is free and open source web-based bug-tracking software that is
+developed by an active group of volunteers in the Mozilla community, and
+used by thousands of projects and companies around the world.
+
+----
+|homepage| |documentation| |irc| |docker_pulls| |license|
+----
+
+Usage
+=====
+
+Bugzilla requires Perl 5.14 (or newer) and tools to compile and install perl
+modules, including make(1) and a C compiler.
+
+For Developers
+--------------
+
+If you have the `cpanm <https://github.com/miyagawa/cpanminus#installation>`__ script in your $PATH,
+Bugzilla can be up and running in three commands.
```

Comment:
Actually not. You need to install certain development tools such as make and gcc among others. So you may need to add a blurb about 'yum groupinstall "Development Tools"' and equivalent for Ubuntu based systems.